screwdriver/scripts/settings-publish.xml

```diff
@@ -1,9 +1,9 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
```

Comment:
directory is incorrect should be `screwdriver/settings/settings-publish.xml` instead of `screwdriver/scripts/settings-publish.xml`